ci/docker_build_and_fuzzit.sh

```diff
@@ -40,11 +41,11 @@ cp ./relwithdebinfo/arrow-ipc-fuzzing-test ./relwithdebinfo/out/fuzzer
 ldd ./relwithdebinfo/arrow-ipc-fuzzing-test | grep "=> /" | awk '{print $3}' | xargs -I '{}' cp -v '{}' ./relwithdebinfo/out/.
 cd ./relwithdebinfo/out/
 tar -czvf fuzzer.tar.gz *
+stat fuzzer.tar.gz
 cd ../../
 
 export TARGET_ID=u79f6bXYgNH4NkU99iWK
```

Comment:
@pitrou this is no longer necessary. Your new account is located at https://app.fuzzit.dev/orgs/apache-arrow/dashboard